cmd/juju/model/destroy.go

```diff
@@ -101,18 +101,33 @@ const destroyExamples = `
 `
 
 var destroyModelMsg = `
-This command will destroy the %q model and all its resources. It cannot be stopped.`[1:]
+This command will destroy the %q model and affect the following resources. It cannot be stopped.`[1:]
 
 var destroyModelMsgDetails = `
 {{- if gt .MachineCount 0}}
- - {{.MachineCount}} {{if .IsCaaS}}container{{else}}machine{{end}}{{if gt .MachineCount 1}}s{{end}} will be destroyed
-  - {{if .IsCaaS}}container{{else}}machine{{end}} list:{{range .MachineIds}} "{{.}}"{{end}}
+ - {{.MachineCount}} machine{{if gt .MachineCount 1}}s{{end}} will be destroyed
+  - machine list:{{range .MachineIds}} "{{.}}"{{end}}
+{{- end}}
  - {{.ApplicationCount}} application{{if gt .ApplicationCount 1}}s{{end}} will be removed
  {{- if gt (len .ApplicationNames) 0}}
   - application list:{{range .ApplicationNames}} "{{.}}"{{end}}
  {{- end}}
  - {{.FilesystemCount}} filesystem{{if gt .FilesystemCount 1}}s{{end}} and {{.VolumeCount}} volume{{if gt .VolumeCount 1}}s{{end}} will be {{if .ReleaseStorage}}released{{else}}destroyed{{end}}
-{{- end}}
+`
+
+var persistentStorageErrorMsg = `cannot destroy model "{{.ModelName}}"
```

Comment:
Agree this is now getting messy, and really, it doesn't justify using a template.
It's 99% fixed text plus model name and a storage message. We should just use Sprintf with 2 %s placeholders for model name and storage message. The storage message would be composed using go logic and then applied to the printf.

Reply:
Agreed.  Changed to `sprintf`.

Simplified the output a bit:
* filesystem and volume are on different lines, so we don't need an if statement putting "and" inbetween
* rather than having an if statement for plural (if count > 1: add an 's') it now just prints `X volume(s)`

I'll change those back if you like, but they seemed superfluous